test/Scoop-Linting.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+$repo_dir = (Get-Item $MyInvocation.MyCommand.Path).directory.parent.FullName
+$scoop_modules = Get-ChildItem $repo_dir -Filter "*.ps1" -Recurse
```

Comment:
You might want to consider using a `"*.ps*1` filter to also include `.psm1` and `.psd1` files (there are even specific rules just for manifests)

Reply:
Thank you for pointing it out that.